schema/dataKeys.json

```
@@ -132,8 +136,12 @@
           "description": "The product stream e.g. RHEL-tp1"
         },
         "cpe": {
-          "type": "string",
-          "description": "The product CPE ID e.g. cpe:/a:example:openstack:el8"
+          "type": ["array", "string"],
+          "minItems": 1,
+          "description": "The list of product CPE IDs e.g. [cpe:/a:example:openstack:el8]",
```

It probably doesn't matter much, but the example should have `""` for the string. It's not great because they need to be escaped, but still probably better than not have them.
```suggestion
          "description": "The list of product CPE IDs e.g. [\"cpe:/a:example:openstack:el8\"]",
```
I tried running `check-jsonschema` with this and it worked for me.

---

Good point, added